### ISSUE TITLE
Plan a manual data overlay to city API data

### ISSUE THREAD

**Issue by sanjuroj** [hackoregon/transportation-backend#23]
One of the problems with the city's APIs is that they are often lacking date information, or the date information present is non-sensical (e.g. completion dates for uncompleted projects).  We'd like to provide an input page that PBOT can use to manually update the data provided in the API.  

The overlay will probably be saved as a separate table or set of tables.  Then, either on the front-end or back-end, we will need to see if an city API data point has been superseded by a manually entered one and deliver the appropriate data point.

**Comment by znmeb** [comment]
Is the "non-sensical" part definable as a set of input validation tests? If so, can we generate a pre-populated form for the city that says, "These are the records / fields that need to be checked for validity and possibly updated?"

**Comment by sanjuroj** [comment]
Yes, I think that's the right road to follow.  We'll need their help to figure out what that minimum set is though.  It might be nice to build a demo with a very small set (maybe just dates). We can then ask them what the next steps for the form would be (i.e. design, content, etc...)
